docs/migration.rst

```diff
@@ -148,6 +148,22 @@ The :meth:`newton.ModelBuilder.add_joint_free()` method now initializes the posi
 
 The universal and compound joints have been removed in favor of the more general D6 joint.
 
+**Terminology Change: Environment → World**
+
+Newton now uses "world" terminology instead of "environment" for entity grouping in multi-world simulations:
+
++--------------------------------------------------------+------------------------------------------------------------------------+
+| **Previous**                                           | **Current**                                                            |
++--------------------------------------------------------+------------------------------------------------------------------------+
+| ``ModelBuilder.current_env_group``                     | :attr:`newton.ModelBuilder.current_world`                              |
++--------------------------------------------------------+------------------------------------------------------------------------+
+| ``ModelBuilder.add_builder(..., environment=0)``       | ``ModelBuilder.add_builder(..., world=0)``                             |
++--------------------------------------------------------+------------------------------------------------------------------------+
+| ``Model.shape_group``                                  | :attr:`newton.Model.shape_world`                                       |
```

Comment:
We did not have shape groups before in warp.sim, so you can remove those changes from the migration guide.